styles.css

```diff
@@ -656,4 +766,62 @@ header.active .navigations {
   .in-touch {
     margin-left: 75px;
   }
+
+  .pop {
+    display: none;
+    position: fixed;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    top: 0;
+    bottom: 0;
+    max-height: 100vh;
+    width: 100%;
+    overflow: auto;
+    background: rgb(193 199 208 / 50%);
+    backdrop-filter: blur(3px);
+    padding: 50px 16px;
+  }
+
+  .pop-content {
+    padding: 12px;
+    margin: 0 95px;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the popup has a blur effect on its top. This will make your portifolio have a professional UI ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233770355-ac71b666-6cea-4bc5-a9e4-3fe8876bb521.png)